guides/security/BYOD.md

```diff
@@ -0,0 +1,101 @@
+
+This document highlights things you should think about, and how you manage your own device while working on Made Tech products and services. Not only the data you work with but also the software you have installed and how you protect it - like antivirus software.
+
+This only applies to you if you’re using your own device for Made Tech work - including checking your Made Tech email. Devices aren’t just desktop and laptops - they include, for example, tablets and smartphones.
+
+## Using and managing your device
+
+If you’re using your own device for Made Tech work, it means you’re being trusted to use Made Tech information appropriately and keep any Made Tech information or data you hold, or can access, safe.
+
+### Don’t ‘jailbreak’ your device
+‘Jailbreaking’ is the process of removing hardware and software vendor restrictions for the purpose of custom modifications (for example, installing unapproved applications or changing the user interface in an unsupported way).
+
+You must not use a jailbroken device for any Made Tech work or to handle any Made Tech data, including logging into Made Tech services which include (but is not limited to) G-Suite, Slack, password management software, etc.
+
+### Keep your software up to date
+You must keep your software up to date. Old versions of software are often vulnerable to attack. Update all the software on your device whenever a new version is available. This includes:
+
+- Your device’s operating system
+- Web browsers
+- Extensions
+- Plug-ins
+
+You must update your software within 14 days of a new verison being released.
+
+### Use endpoint protection software
+Unless your device is operating iOS or ChromeOS, your device must have endpoint protection software installed, which must also be kept up to date. 
+
+Endpoint protection is more than just antivirus software, and usually includes a firewall and may have other tools to protect your device, like network threat detection.
+
+Any device you plug into the device, like an external USB device, must also be scanned for viruses. We recommend doing a full scan at least once a month.
+
+### Set a password/passphrase
+Your device must require a password/passphrase to login, return from screensaver or wake from sleep. Ideally your device will lock the screen (or start the screensaver) after a few minutes (no more than 5) of the device being idle
+
+### How you manage passwords/passphrases
+Your password/passphrase must conform to the Made Tech Password Policy which will be linked below. You should also use a password manager and 2FA.
+
+### How you connect your device to networks
+We have our own VPN which automatically authenticates with G Suite. Everyone can use the service (not only engineers) for a more secure experience (i.e. protecting yourself when using a public WiFi). To connect to the VPN, please refer to the [guide found in the Handbook.](https://github.com/madetech/handbook/tree/master/guides/vpn) 
+
+### How you store & manage data
```

Comment:
I'm not sure what this section achieves.

I think maybe we should rephrase this about being required to store data on backed up cloud services (G-Suite or GitHub)

Reply:
Oh, that's already covered below. In which case I'm not too sure what this section covers

Reply:
It was in the old approved PR so I copied it over. 

I'm not 💯 as to whether you are referring to the VPN snippet of the storing of data

Reply:
The 'store and manage data' section. I'm not convinced it's imparting any useful information. I'm not sure I'd consider do anything different after reading this, it's just telling me data can be lost or damaged.

Reply:
Have deleted this section